index.html

```diff
@@ -35,6 +35,125 @@ <h2 class="greeting-intro">
+                <h4 class="works-title">Multi-Post Stories Gain + Glory</h4>
+                <ul class="skill-wrap works-skills">
+                    <li class="work-skill">Ruby on rails</li>
+                    <li class="work-skill">css</li>
+                    <li class="work-skill">JavScript</li>
+                    <li class="work-skill">html</li>
+                </ul>
+                <button class="home-btn work-detail-button">See Project</button>
+            </section>
+        </section>
+        <section class="works-card">
+            <section class="works-card-inner">
+                <h4 class="works-title">Multi-Post Stories Gain + Glory</h4>
+                <ul class="skill-wrap works-skills">
+                    <li class="work-skill">Ruby on rails</li>
+                    <li class="work-skill">css</li>
+                    <li class="work-skill">JavScript</li>
+                    <li class="work-skill">html</li>
+                </ul>
+                <button class="home-btn work-detail-button">See Project</button>
+            </section>
+        </section>
+        <section class="works-card">
+            <section class="works-card-inner">
+                <h4 class="works-title">Multi-Post Stories Gain + Glory</h4>
+                <ul class="skill-wrap works-skills">
+                    <li class="work-skill">Ruby on rails</li>
+                    <li class="work-skill">css</li>
+                    <li class="work-skill">JavScript</li>
+                    <li class="work-skill">html</li>
+                </ul>
+                <button class="home-btn work-detail-button">See Project</button>
+            </section>
+        </section>
+        <section class="works-card">
+            <section class="works-card-inner">
+                <h4 class="works-title">Multi-Post Stories Gain + Glory</h4>
+                <ul class="skill-wrap works-skills">
+                    <li class="work-skill">Ruby on rails</li>
+                    <li class="work-skill">css</li>
+                    <li class="work-skill">JavScript</li>
+                    <li class="work-skill">html</li>
+                </ul>
+                <button class="home-btn work-detail-button">See Project</button>
+            </section>
+        </section>
+        <section class="works-card">
+            <section class="works-card-inner">
+                <h4 class="works-title">Multi-Post Stories Gain + Glory</h4>
+                <ul class="skill-wrap works-skills">
+                    <li class="work-skill">Ruby on rails</li>
+                    <li class="work-skill">css</li>
+                    <li class="work-skill">JavScript</li>
+                    <li class="work-skill">html</li>
+                </ul>
+                <button class="home-btn work-detail-button">See Project</button>
+            </section>
+        </section>
+    </section>
+    <section class="about-wrapper">
+        <section class="about-header">
```

Comment:
Kindly. the semantic SECTION tags are used to identify each theme of your page, I recommend you use them like this, inside them you can put the necessary content, but putting a SECTION inside another SECTION is not a good practice, I recommend changing it to a DIV.

Reply:
This recommendation has been adopted. Thanks alot